autoemulate/compare.py

```diff
@@ -368,6 +378,74 @@ def load_model(self, path=None):
 
         return serialiser._load_model(path)
 
+    def print_settings(self):
```

Comment:
This works/looks great.

Reply:
One thought: should we rename this to `print_setup` or do you have a reason to call this `settings` ?

Reply:
Great point!

Reply:
Addresed in https://github.com/alan-turing-institute/autoemulate/pull/195/commits/75d5ae61506dd76c7dd2ad3157c3cdc05c4e19e9